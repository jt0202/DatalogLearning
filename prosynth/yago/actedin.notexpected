sridevi	harsh_times
aadhi_actor	the_keys_to_the_house
ann_sothern	unlawful_entry_film
bruce_le	peege
nathan_lane	frankenstein_1994_film
