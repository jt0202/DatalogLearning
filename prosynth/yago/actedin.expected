john_lennon	imagine_film
kamal_haasan	panchathantiram
john_cleese	the_strange_case_of_the_end_of_civilization_as_we_know_it
eddie_griffin	dysfunktional_family
bob_saget	becoming_dick
rodney_dangerfield	rover_dangerfield
al_pacino	chinese_coffee
lucas_belvaux	nature_contre_nature
martin_lawrence	you_so_crazy
eddie_murphy	beverly_hills_cop_ii
emilio_estevez	bobby_2006_film
rodney_dangerfield	back_by_midnight
dane_cook	tourgasm
jill_hennessy	the_acting_class
kamal_haasan	manmadhan_ambu
burt_reynolds	the_man_from_left_field
chris_rock	i_think_i_love_my_wife
alan_alda	the_four_seasons_film
tamer_hosny	omar__salma_2
palito_ortega	amigos_para_la_aventura
kenneth_branagh	peters_friends
robin_williams	mrs_doubtfire
chris_rock	good_hair
warren_beatty	heaven_can_wait_1978_film
eddie_murphy	another_48_hrs
lou_diamond_phillips	sioux_city_film
emilio_estevez	the_war_at_home_film
denis_leary	favorite_deadly_sins
jennifer_westfeldt	kissing_jessica_stein
kenneth_branagh	hamlet_1996_film
sunil_dutt	daaku_aur_jawan
lou_diamond_phillips	dangerous_touch
liza_minnelli	liza_with_a_z
paul_spence	fubar_film
harpo_marx	love_happy
w_c_fields	the_bank_dick
quentin_tarantino	from_dusk_till_dawn
pierre_andre	jangan_pandang_belakang
kamal_haasan	thevar_magan
warren_beatty	love_affair_1994_film
sunil_dutt	yaadein_1964_film
lillete_dubey	lt30_Days_in_September_Playgt
kamal_haasan	dasavathaaram
tamer_hosny	noor_eieny
john_cleese	the_human_face
kamal_haasan	virumaandi
alex_chandon	chainsaw_scumfuck
john_wayne	the_longest_day_film
kamal_haasan	pushpak
bruce_willis	live_free_or_die_hard
madonna_entertainer	im_going_to_tell_you_a_secret
quentin_tarantino	my_best_friends_birthday
craig_ferguson	ill_be_there_film
doctor_dré	whos_the_man
tom_waits	big_time_1988_film
emilio_estevez	wisdom_film
frank_buck_animal_collector	wild_cargo_1934_film
cliff_robertson	the_pilot_film
eddie_murphy	coming_to_america
maija_digiorgio	bitter_jester
warren_beatty	bulworth
al_pacino	looking_for_richard
kenneth_branagh	dead_again
darren_dalton	the_day_the_earth_stopped
yılmaz_güney	umut
kenneth_branagh	henry_v_1989_film
laura_kirk	lisa_picard_is_famous
john_cleese	wine_for_the_confused
lucas_belvaux	cavale
denis_leary	rescue_me_tv_series
jimmy_kimmel	windy_city_heat
jerry_seinfeld	bee_movie
spike_milligan	ghost_in_the_noonday_sun
frank_buck_animal_collector	jungle_cavalcade_1941_film
marty_feldman	in_god_we_trut
burt_reynolds	sharkys_machine_film
emma_thompson	nanny_mcphee
john_wayne	the_undefeated_1969_film
paul_spence	fubar_2_film
